src/constants/indicators.js

```diff
@@ -431,15 +431,15 @@ export default {
   newBusinessEntrants: {
     name: 'New business entry rate',
     legendText: 'Entrants',
-    format: FORMATS.THREE_DECIMALS,
+    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
     contextual: false,
     lastUpdated: '2016',
     source: 'New business entrants and exits, New Businesses as a share of the stock',
   },
   newBusinessExits: {
     name: 'Share of ceased businesses',
     legendText: 'Exits',
-    format: FORMATS.THREE_DECIMALS,
+    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
```

Comment:
Minor: same comment as above, if these are forced to at least one decimal, the chart y axis looks a bit neater.